budget/web.py

```diff
@@ -119,13 +150,21 @@ def authenticate(project_id=None):
 
 @main.route("/")
 def home():
-    project_form = ProjectForm()
+    # If ADMIN_PASSWORD is empty we consider that we are not in admin mode
+    admin = bool(current_app.config['ADMIN_PASSWORD'])
```

Comment:
I honestly find that this logic is a bit odd — Maybe this is because of the naming. We might want to name the `admin` variable something else, like `is_admin_mode_enabled` or `admin_mode`.